module/dkg/broker.go

```diff
@@ -241,30 +257,36 @@ func (b *Broker) Poll(referenceBlock flow.Identifier) error {
 	for _, msg := range msgs {
 		ok, err := b.verifyBroadcastMessage(msg)
 		if err != nil {
+			// CAUTION: potential slashable offense
```

Comment:
This function seems to only authenticate the message. Failing to authenticate the message can't be slashable.

Reply:
Good point